From 8f62821384c0577aa0f629d73fae5409a486e05b Mon Sep 17 00:00:00 2001
From: Christian Jaeger <ch@christianjaeger.ch>
Date: Thu, 10 Dec 2020 04:04:29 +0000
Subject: .bashrc 001: [PATCH 086/191] .bashrc: remove stale commented lines

Originally from someone else's config.

This concludes the removal of the use of the `alias` shell builtin.
---
 .bashrc | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/.bashrc b/.bashrc
--- a/.bashrc
+++ b/.bashrc
@@ -40,8 +40,6 @@ case "$TERM" in
 if [ "$TERM" != "dumb" ]; then
     eval "$(dircolors -b)"
     ls () { command ls --color=auto "$@"; }
-    #alias dir='ls --color=auto --format=vertical'
-    #alias vdir='ls --color=auto --format=long'
 fi
 
 # enable programmable completion features (you don't need to enable
